libraries/ms-common/src/x86/gpio.c

```diff
@@ -29,10 +29,12 @@ static uint32_t prv_get_index(const GPIOAddress *address) {
 }
 
 StatusCode gpio_init(void) {
-  GPIOSettings default_settings = { .direction = GPIO_DIR_IN,
-                                    .state = GPIO_STATE_LOW,
-                                    .resistor = GPIO_RES_NONE,
-                                    .alt_function = GPIO_ALTFN_NONE };
+  GPIOSettings default_settings = {
+    .direction = GPIO_DIR_IN,
```

Comment:
This isn't possible so far as I can tell, the rule is one per line only if it doesn't fit on one line otherwise put it on one line. Seems to be fixed with that rule and I tried my best to avoid it without any luck.